tests/test_examples.py

```diff
@@ -133,26 +133,6 @@ def get_mol_name_from_pkl(fname):
     return dG_preds, dG_expts
 
 
-@pytest.mark.skip("needs update since removal of lambda dependence in nonbonded potentials")
```

Comment:
@maxentile this seemed stale, but wasn't sure; let me know if not and I will remove from the diff.